### PR TITLE
(FACT-2219) augeas fact for AIX

### PR DESCRIPTION
**Pull request by ciprianbadescu**

### CODE REVIEW

File: lib/resolvers/os_level_resolver.rb [hunk 1]
```diff
@@ -18,8 +18,9 @@ def resolve(fact_name)
         end
 
         def read_oslevel(fact_name)
-          output, _status = Open3.capture2('oslevel -s')
+          output, _status = Open3.capture2('/usr/bin/oslevel -s 2>/dev/null')
           @@fact_list[:build] = output
+          @@fact_list[:kernel] = 'AIX'
```
Comment (gimmyxd) [inline lib/resolvers/os_level_resolver.rb]: should this be hardcoded to AIX?

Comment (ciprianbadescu) [reply lib/resolvers/os_level_resolver.rb]: This is my understanding from C facter: https://github.com/puppetlabs/facter/blob/master/lib/src/facts/aix/kernel_resolver.cc#L20

File: spec/facter/resolvers/augeas_resolver_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+describe 'OsLevel' do
```
Comment (oanatmaria) [inline spec/facter/resolvers/augeas_resolver_spec.rb]: describe augeas resolver

Comment (ciprianbadescu) [reply spec/facter/resolvers/augeas_resolver_spec.rb]: solved

**Review by oanatmaria — APPROVED** [review]

**Pull request merged**